pom.xml

```diff
@@ -189,6 +189,47 @@
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>org.revapi</groupId>
+          <artifactId>revapi-maven-plugin</artifactId>
```

Comment:
Can't we move this config into kie-parent - presume it will be the same for kie-api etc.